public/placeholder.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="23.5" height="30.821" viewBox="0 0 23.5 30.821"><defs><style>.a{fill:#484848;}</style></defs><g transform="translate(-76)"><g transform="translate(76)"><g transform="translate(0)"><path class="a" d="M87.75,0C81.271,0,76,4.861,76,10.836a10.2,10.2,0,0,0,1.754,5.7l9.327,13.861a1,1,0,0,0,.832.427h.008a1,1,0,0,0,.832-.439l9.089-14A10.193,10.193,0,0,0,99.5,10.836C99.5,4.861,94.229,0,87.75,0Zm8.412,15.46L87.9,28.184l-8.48-12.6a8.507,8.507,0,0,1-1.473-4.746c0-4.979,4.406-9.042,9.8-9.042s9.8,4.063,9.8,9.042A8.5,8.5,0,0,1,96.162,15.46Z" transform="translate(-76)"/></g></g><g transform="translate(81.875 5.418)"><g transform="translate(0)"><path class="a" d="M171.875,90a5.436,5.436,0,1,0,5.875,5.418A5.667,5.667,0,0,0,171.875,90Zm0,9.042a3.635,3.635,0,1,1,3.923-3.624A3.789,3.789,0,0,1,171.875,99.042Z" transform="translate(-166 -90)"/></g></g></g></svg>
```

Comment:
What's the licensing on these SVGs? Did you make them yourself?

Reply:
Not sure, I took them from the mockup. @positiveimpact ?

Reply:
All the icons are from flaticon.com. When we launch, we'll have either to pay a monthly base fee of 7.50€ for using all the icons or credit the author of each icon. Alternatively, I could also look for a resource for fully free icons.

Reply:
I've had a look at free icons and couldn't find many good ones. However, if we have to credit multiple authors, our footer will become really packed. Most icons I used are by https://www.flaticon.com/authors/smashicons. I suggest we credit them in the footer and I make sure all icons are by this author.

Reply:
I have a preference for free icons. Have we tried icons from Fork Awesome, or IconFinder, or IconMonstr, or just using emojis?

Reply:
Thanks for the suggestions, I'll try if I can switch out all the icons with free icons!